ember-toucan-core/src/components/form/input-field.hbs

```diff
@@ -0,0 +1,32 @@
+<div
+  class="flex flex-col {{if @isDisabled 'text-disabled cursor-not-allowed'}}"
+>
+  <Form::Field as |field|>
+    <field.Label for={{field.id}}>{{@label}}</field.Label>
+
+    {{#if @hint}}
+      <field.Hint id={{field.hintId}}>{{@hint}}</field.Hint>
+    {{/if}}
+
+    <field.Control class="flex">
+      <input
+        class="m-1 h-6 w-0 flex-grow bg-transparent p-1
```

Comment:
Do we want the margin here via `m-1`? That'll push elements around a bit.

I tried getting _pretty_ close to what we are going for in with the textarea, which I think is what we'll mostly want here https://github.com/CrowdStrike/ember-toucan-core/pull/36/files#diff-0cf8ecb43c00a4f64c9e8710b853499367e0130793dffdb772c0bb4ef9a4dd0eR13


Something like this may be close:
```hbs
class="block h-6 p-1 bg-overlay-1 focus:outline-none rounded-sm shadow-focusable-outline focus:shadow-focus-outline {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}"
```

I also left off other soon™ disabled styles, but happy to align with whatever so we remain consistent as to make it easier on us later when we go and update the styles to match post-testing!